src/utils/ParseRouteUtils.js

```diff
@@ -676,14 +676,17 @@ function parseRoutes(
         const walkDepartureTime = convertMillisecondsToISOString(startTimeMs);
         const walkArrivalTime = convertMillisecondsToISOString(endTimeMs);
 
-        directions[0].startTime = walkDepartureTime;
-        directions[0].endTime = walkArrivalTime;
+        const walkDirections = {
+          ...directions[0],
```

Comment:
out of curiosity what does the `...direction[0]` do/mean

Reply:
directions is an array and walking will always have 1 direction in directions

Reply:
lmao i was referring to the `...` but Dr Chan explained it to me so we chillin. tyty